plugins/Goals/Controller.php

```diff
@@ -251,6 +252,10 @@ public function getEvolutionGraph(array $columns = array(), $idGoal = false, arr
         $langString = $idGoal ? 'Goals_SingleGoalOverviewDocumentation' : 'Goals_GoalsOverviewDocumentation';
         $view->config->documentation = $this->translator->translate($langString, '<br />');
 
+        if ($view instanceof Evolution) {
+            $view->requestConfig->request_parameters_to_modify['format_metrics'] = 0;
+        }
```

Comment:
Actually I was wondering why only the Average Order Value is broken, but the problem seems that that metric is added as processed metric, while all other metrics are "normal" metrics, which causes the formatter to format only that one.
Guess that's yet another issue of not having a clean metric system, where all metrics in a datatable have a metric class, that might format if needed or not.
Might be something we could try to improve for Matomo 5...

Reply:
Definitely a good candidate for Matomo 5. There seem to be a lot of issues at the moment caused by formatted/unformatted metrics, we've fixed quite a few but the whole system would benefit from some consistency :+1: